docs/operator/configuration_parameters.md

```diff
@@ -532,36 +532,48 @@ and only contains information about the specific RSE's for the Rucio instance.
 Read more about RSEs [here](../started/concepts/rucio_storage_element.md)
 and how to set them up [here](./configuration.md#creating-new-rses).
 
-- **associated_sites**: <!--??--> Separated by commas. Default: `None`.
-- **auto_approve_bytes**: Auto Approve Limit. Example: `500GB`. No default.
+- **associated_sites**: <!--??--> String. Separated by commas. Used for chaining of subscriptions so that transfers to one RSE will also be mirrored to associated_sites. No default.
+- **archive_timeout**: <!--??--> Integer: For transfers with a tape destination, control the number of seconds Rucio will wait for the tape archival of the file to complete. No default.
+- **auto_approve_bytes**: <!--??--> Auto Approve Limit. Example: `500GB`. No default.
 - **auto_approve_files**: <!--??--> No default.
-- **available_for_multihop**: Boolean. If True, allow to use this RSE as an
-  intermediate hop in a multi-hop transfer. Default: `False`.
+- **available_for_multihop**:  <!--??--> Boolean. If True, allow to use this RSE as an intermediate hop in a multi-hop transfer. Default: `False`.
 - **block_manual_approval**: <!--??--> Boolean. Default: `False`.
-- **country**: <!--??--> No default.
-- **greedyDeletion**: <!--??--> Boolean. Default: `False`.
+- **bittorrent_tracker_addr**: <!--??--> String. Used to configure the URL of the bittorrent tracker API when using the torrent transfer manager. No Default.
+- **checksum_key**: <!--??--> String. Used to specify an alternate RSE attribute to search for supported checksums beyond those with global support (ADLER32, MD5). Default: `supported_checksums`.
+- **country**: <!--??--> String. Used for country level granularity of RSE selectors. No default.
+- **driver_name_rse_attribute**: <!--??--> String. Used to specify alternate drivers when using the Bittorrent transfer manager. Default: `bittorrent_driver`.
+- **fts**: <!--??--> String. Specify the REST API URL of the FTS3 transfer manager. No default.
+- **greedyDeletion**: <!--??--> Boolean. Control if files on an RSE with no rule locking them there should be deleted by the Reaper. Default says that files should only be deleted when additional space on the RSE is required. Default: `False`.
+- **group_by_rse_attribute**: <!--??--> String. Control the RSE attribute (such as `country`) which transfer source RSEs will be grouped by when determining an appopriate transfer source. Default: `UNKNOWN`.
```

Comment:
That's difficult to do, as a site can pick it's own additional RSE attributes to group by. Even `country` for example isn't actually required, and it could be anything. For example DUNE uses a `US_SITES: True` and `SITES: US_FNAL-Fermigrid` attributes, but it could be anything even as far-fetched as `I_LOVE_CATS: Yup-I-Do`.